packages/docs/components/SideNav/SideNav.tsx

```diff
@@ -62,14 +63,14 @@ export const SideNav: React.FC = () => {
           <SideNavLink href="/Panel/PanelPage" as="/panel">
             Panel
           </SideNavLink>
-          <SideNavLink href="/Table/TablePage" as="/table">
-            Table
-          </SideNavLink>
           <SideNavLink href="/StatefulTable/StatefulTablePage" as="/statefulTable">
-            StatefulTable
+            Stateful table
```

Comment:
Can you touch upon the reasoning for this change? I get the thought process, but this might be a weird change for developers who are used to seeing the component name (i.e. `StatefulTable`)

Reply:
Yeah I can see that too… my thinking was aligning sidebar links and page titles. What's your recommendation?

Reply:
My recommendation, for now, is to keep it as the component name since this is still developer documentation. Later, when we add the design documentation, we can "normalize" these.

Reply:
Lets do that then